meson.build

```
@@ -34,10 +35,11 @@ elif cc.get_id() == 'msvc'
           'when building with MSVC')
   endif
 endif
+if not cy.version().version_compare('>=0.29.34')
+  error('SciPy requires Cython >= 0.29.34')
```

Is there a "project_name" macro we could use here instead?

```suggestion
  error('NumPy requires Cython >= 0.29.34')
```

---

Good catch, thanks. I'll see if the current CI run passes, then will push this with a few more tweaks.

> Is there a "project_name" macro we could use here instead?

Not that I know of - but seems more complexity than it's worth anyway, and makes it harder to grep for the error string, so I'd prefer not to do that.